package.json

```diff
@@ -37,7 +37,9 @@
   },
   "scripts": {
     "test": "ln -sf ./build-tmp-napi-v3 ./build && ./node_modules/mocha/bin/mocha.js --timeout 5s",
-    "install": "node-pre-gyp install --fallback-to-build"
+    "install": "node-pre-gyp install --fallback-to-build",
+    "build": "npm install --build-from-source",
+    "build-release": "./node_modules/.bin/node-pre-gyp rebuild --production && ./node_modules/.bin/node-pre-gyp package"
```

Comment:
We have encountered compatibility issues before. So we may need to test the build with all supported Node.js versions before release the new version. 
For example: 
```nodever 16 && npm test test/dataTypes.js && nodever 14 && npm test test/dataTypes.js && nodever 12 && npm test test/dataTypes.js```

Reply:
Because the package is built with Node-API, the only compatibility that _should_ matter is the NAPI version. In package.json, it looks like this package only build for v3:

https://github.com/IBM/nodejs-idb-connector/blob/99e127a1c3eedb31cdb858fd911411a4d5d2d1ef/package.json#L63-L65

No matter what version of Node.js we build with, it should just built a `napi3` binary that can be used with any version of Node.js that supports at least that version, which is every version of Node since 8.11.2 and 10.16.0 (and every version of 12+): https://nodejs.org/api/n-api.html#node-api-version-matrix